packages/components/src/search/index.js

```diff
@@ -221,12 +232,14 @@ Search.propTypes = {
 	 */
 	placeholder: PropTypes.string,
 	/**
-	 * An array of objects describing selected values.
+	 * An array of objects describing selected values. If the label of the selected
+	 * value is omitted, the Tag of that value will not be rendered inside the
+	 * search box.
```

Comment:
Is this so that search terms found in things like email are not rendered?

Reply:
For the table search box, we need to pass a `selected` prop so the selected elements are excluded from the autocompleter suggestions, but we don't want them to be displayed in `<Tag>`s because they are already shown in the filters search box above. That's why I added the possibility of passing `selected` items that will be hidden.

> Just the comment about labels on tags. If I understand correctly, if the label is undefined, then a Tag is created, but it won't get rendered.

Not really, the `Tag` for selected items without label should not be created at all. See:

https://github.com/woocommerce/wc-admin/pull/1085/files#diff-9d5e585ab84ca2db27f702f4836fe65cR97